core/src/main/java/org/radargun/stages/test/LoadStage.java

```diff
@@ -0,0 +1,129 @@
+package org.radargun.stages.test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.radargun.DistStageAck;
+import org.radargun.config.Property;
+import org.radargun.config.Stage;
+import org.radargun.stages.AbstractDistStage;
+import org.radargun.utils.TimeConverter;
+
+/**
+ * @author Radim Vansa &lt;rvansa@redhat.com&gt;
+ */
+@Stage(doc = "Manages thread for data loading")
+public abstract class LoadStage extends AbstractDistStage {
+   @Property(doc = "Number of loaded entries after which a log entry should be written. Default is 10000.")
+   protected long logPeriod = 10000;
+
+   @Property(doc = "The number of threads that should load the entries on one slave. Default is 10.")
+   protected int numThreads = 10;
+
+   @Property(doc = "Seed used for initialization of random generators - with same seed (and other arguments)," +
+         " the stage guarantees same entries added to the cache. By default the seed is not set.")
+   protected Long seed;
+
+   @Property(doc = "During loading phase, if the insert fails, try it again. This is the maximum number of attempts. Default is 10.")
+   protected int maxLoadAttempts = 10;
```

Comment:
Well spotted, I forgot to remove them from LoadDataStage (I'll probably rename that to LoadCacheStage)